packages/core/src/desmosclient.ts

```diff
@@ -467,4 +471,35 @@ export class DesmosClient extends SigningStargateClient {
       }),
     };
   }
+
+  public async sendIbcTokens(
```

Comment:
I would name this `sendIBCTokens`. But why do we need this anyway? Currently the `DesmosClient` does not expose methods that allow creating Desmos-specific transactions. Why would we want to include this?

Reply:
This was already present since `DesmosClient` was extending `SigningStargateClient`. This is needed to ensure backword compatibility.

Reply:
Ok, can you add a comment so we can remember it's for backward compatibility?